### DISCUSSION TITLE
LinterCop not working

### DISCUSSION THREAD

**Discussion by MOomenAL** [StefanMaron/BusinessCentral.LinterCop#219] [Q&A] [answered]
Hello Stefan,

First of, thanks for all your efforts.
Based on the BC Techdays I am interested to see what kind of value LinterCop can add any value to our dev department. 
But unfortunately I do not seem to get it to work; 
- I've read the readme on Github. 
- The cop shows in my VS Code as active. 
- Ruleset has been added. 

What advice can you give to me find out what is going wrong? Or perhaps someone else who can tell me more? Thanks in advance.

Regards,
Martin Oomen

**Comment by StefanMaron** [comment, marked as answer]
Did you add the linter to the Code analyzers in vs code settings?
You also want to make sure code analysis is enabled

``` Json
"al.codeAnalyzers": [
	"${CodeCop}",
	"${UICop}",
	"${AppSourceCop}",
	"${analyzerFolder}BusinessCentral.LinterCop.dll"
],
"al.enableCodeAnalysis": true,
```

**Reply by MOomenAL** [reply]
Thank you for your reply. No, this was not the case, but it did help me find it. Apparantly the _al.backgroundCodeAnalysis_ was set to false earlier, switching it to true solves my problem.
